src/renderers/native-rt/ReactNativeRTFiberRenderer.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule ReactNativeRTFiberRenderer
+ * @flow
+ */
+
+'use strict';
+
+const ReactFiberReconciler = require('ReactFiberReconciler');
+const ReactNativeRTComponentTree = require('ReactNativeRTComponentTree');
+const ReactNativeRTTagHandles = require('ReactNativeRTTagHandles');
+const RTManager = require('RTManager');
+
+const emptyObject = require('fbjs/lib/emptyObject');
+const invariant = require('fbjs/lib/invariant');
+
+export type Container = number;
+export type Instance = number;
+export type Props = Object;
+export type TextInstance = number;
+
+const {precacheFiberNode, updateFiberProps} = ReactNativeRTComponentTree;
+
+const NativeRTRenderer = ReactFiberReconciler({
+  appendChild(parentInstance: Instance, child: Instance | TextInstance): void {
+    RTManager.appendChild(parentInstance, child);
+  },
+
+  appendChildToContainer(
+    parentInstance: Container,
+    child: Instance | TextInstance,
+  ): void {
+    RTManager.appendChild(parentInstance, child);
+  },
+
+  appendInitialChild(
+    parentInstance: Instance,
+    child: Instance | TextInstance,
+  ): void {
+    RTManager.appendChildToDetachedParent(parentInstance, child);
+  },
+
+  commitTextUpdate(
+    textInstance: TextInstance,
+    oldText: string,
+    newText: string,
+  ): void {
+    invariant(false, 'Text components are not yet supported.');
+  },
+
+  commitMount(
+    instance: Instance,
+    type: string,
+    newProps: Props,
+    internalInstanceHandle: Object,
+  ): void {
+    // Noop
+  },
+
+  commitUpdate(
+    instance: Instance,
+    updatePayload: Object,
+    type: string,
+    oldProps: Props,
+    newProps: Props,
+    internalInstanceHandle: Object,
+  ): void {
+    updateFiberProps(instance, newProps);
+    RTManager.updateNode(instance, newProps);
```

Comment:
Is it okay to always make update calls (even if there are no changes)? We added a check to avoid no-op bridge calls for RN b'c of the expense.

Reply:
We don't have any view config and no structural info at this point. We could maybe do a shallow clone. We probably need to do a pass anyway since the props will contain function pointers that are not serializable. Not sure what we'll do here yet.